CHIPAsia/chip-for-woocommerce
# Version 1.4.6 release

## What does this change?
- Fix issue with WooCommerce Block checkout doesn't load bank list when multiple payment method enabled.
- Change FPX Bank Enquiry API endpoint to https://api.chip-in.asia/health_check/fpx_b2c
- Bump version compatibility and tested with latest version of WordPress and WooCommerce

## Asana / Jira / Trello task link
[Asana](https://app.asana.com/0/1203417845528369/1206127132211956/f)

## How to test
1. Create WooCommerce Block Checkout page
1. Activate multiple payment method (CHIP) with bypass payment page
1. Check WooCommerce Block Checkout where dropdown option should include FPX Bank list

## Images

## PR Checklist:
